src/app/js/formats/istex/fetchIstexData.js

```diff
@@ -1,11 +1,13 @@
 import fetch from '../../lib/fetch';
 import composeAsync from '../../../../common/lib/composeAsync';
+import URL from 'url';
 
 const istexApiUrl = process.env.ISTEX_API_URL;
 
 export const getUrl = ({ props: { resource, field }, page, perPage }) => {
     const value = resource[field.name];
-    const output = 'id,title,publicationDate,fulltext,abstract';
+    const output =
```

Comment:
nice to have: this could be refactored to work like injectData and allow ssr for this format.

Reply:
you're right, but I don't know how to do that, and I have no time to explore this new way